tests/lib/IframeApi.spec.js

```diff
@@ -62,7 +62,7 @@ describe('IframeApi', () => {
         expect(CookieJar.eat).not.toHaveBeenCalled();
         expect(AccountStore.instance.list).not.toHaveBeenCalled();
         expect(KeyStore.instance.list).toHaveBeenCalled();
-        expect(listedKeyObjects).toEqual(Dummy.keyInfoObjects());
+        expect(listedKeyObjects).toEqual(Dummy.keyInfoObjects().reverse());
```

Comment:
why `reverse()` ?

Reply:
Ok, both options are fine, but a comment would be nice here :)

Reply:
I have reversed the DUMMY data objects in a separate branch https://github.com/nimiq/keyguard-next/commit/bffceb3dc3a17e4a0581595fcaf94862ce0e601a to not clutter this PR.

My suggestion is to create a separate PR for that after this has been merged. Do you agree, or should I add it to this PR?

Reply:
Agree :)